testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/api/ClientCompatibilityUnitTestCase.java

```diff
@@ -295,7 +305,6 @@ protected static ModelControllerClient createClient(final String artifact, final
 
         final ChildFirstClassLoaderBuilder classLoaderBuilder = new ChildFirstClassLoaderBuilder(false);
         classLoaderBuilder.addRecursiveMavenResourceURL(artifact + ":" + version, excludes);
-        classLoaderBuilder.addParentFirstClassPattern("org.jboss.as.controller.client.ModelControllerClientConfiguration");
```

Comment:
Why is this removed ?

Reply:
@rsvoboda Because its presence was wrong and results in the test not working. :)  ModelControllerClientConfiguration is not used directly by the test driver and therefore doesn't should not be on the parent-first class path. If it is on the parent first classpath, the legacy client being constructed instead of using the ModelControllerClientConfiguration class that shipped with the legacy driver instead uses the one that's in current WF Core. That results in an incorrectly constructed client that fails.

Reply:
Thanks for sharing details